back/src/queue/jobs/types.ts

```diff
@@ -0,0 +1,3 @@
+import { Mail } from "../../mailer/types";
+
+export type sendMailJobData = { payload: Mail };
```

Comment:
Par convention, on commence habituellement les types par une majuscule
```suggestion
export type SendMailJobData = { payload: Mail };
```

Reply:
Je supprime ce type finalement Mail suffit